spec/requests/create_list_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe 'create list api' do
+  it 'creates a list' do
+
+    user = create(:user)
+    api_key = user.api_key
+
+    post "/api/v1/lists", {"list" => {"title" => "List 1"}}, "X-Api-Key" => api_key
+
+    expect(response.status).to eq(200)
```

Comment:
`201` for "created" would be a more accurate response code for a successful create request.